app/models/scheme.rb

```diff
@@ -1,4 +1,17 @@
 class Scheme < ActiveRecord::Base
   has_many :behaviors
   has_many :subjects
+
+  accepts_nested_attributes_for :behaviors
```

Comment:
These are actually unnecessary, since Rails doesn't like setting the ID on create.